Makefile

```diff
@@ -42,17 +45,28 @@ build-mock-images:
 	docker build -t "test.localhost/amazon/wants-update" -f "scripts/dockerfiles/wants-update.dockerfile" .
 	docker build -t "test.localhost/amazon/exit-success" -f "scripts/dockerfiles/exit-success.dockerfile" .
 
-sources:
+sources.tgz: ecs-init/version/version.go
 	./scripts/update-version.sh
```

Comment:
This `update-version.sh` shouldn't be needed if you're generating it as a dependent target.